src/applications/coronavirus-screener/components/FormResult.jsx

```diff
@@ -7,7 +7,10 @@ import recordEvent from 'platform/monitoring/record-event';
 export default function FormResult({ formState }) {
   let result;
   let resultClass = '';
-  const [resultSubmitted, setResultSubmittedState] = React.useState(false);
+  const [resultSubmitted, setResultSubmittedState] = React.useState({
+    isSubmitted: false,
+    startTime: 0,
```

Comment:
maybe set `startTime`  to `moment()` like https://momentjs.com/docs/#/parsing/now/ and then use `moment.diff()` with that?